File: nvidia_tutorials/prim/example_2.py
'''Defines a Sphere Prim'''

from pxr import Usd, UsdGeom

file_path = "_assets/sphere_prim.usda"
stage: Usd.Stage = Usd.Stage.CreateNew(file_path)
sphere : UsdGeom.Sphere = UsdGeom.Sphere.Define(stage,"/hello")
sphere.CreateRadiusAttr().Set(2)

stage.Save()

File: nvidia_tutorials/prim/example_2/example2.cpp
#include <iostream>
#include <pxr/usd/usd/stage.h>
#include <pxr/usd/usdGeom/xform.h>
#include <pxr/usd/usdGeom/sphere.h>
#include <pxr/usd/sdf/path.h>

int main()
{
  pxr::UsdStageRefPtr stage = pxr::UsdStage::CreateNew("_assets/sphere_prim.usda");
  pxr::UsdGeomSphere sphere = pxr::UsdGeomSphere::Define(stage,pxr::SdfPath("/hello"));
  sphere.CreateRadiusAttr().Set(2.);
  stage->GetRootLayer()->Save();
}

